_maps/shuttles/shiptest/scoutcourier.dmm

```diff
@@ -0,0 +1,2155 @@
+"nF" = (
+/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2{
+	dir = 4
+	},
+/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4{
+	dir = 8
+	},
+/obj/structure/cable{
+	icon_state = "4-8"
+	},
+/turf/open/floor/plasteel/white,
+/area/ship/hallway/central)
+"nO" = (
+/obj/machinery/atmospherics/components/unary/passive_vent/layer4{
+	dir = 1
+	},
+/obj/effect/turf_decal/box/red,
+/obj/machinery/camera,
+/turf/open/floor/plating,
+/area/ship/external)
+"oN" = (
+/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4,
+/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2,
+/obj/machinery/door/airlock{
+	name = "Cryopods"
+	},
+/obj/effect/turf_decal/trimline/opaque/syndiered/filled/warning{
+	dir = 1
+	},
+/obj/effect/turf_decal/trimline/opaque/syndiered/filled/warning,
+/obj/machinery/door/firedoor/border_only{
+	dir = 1
+	},
+/obj/machinery/door/firedoor/border_only,
+/turf/open/floor/plasteel/white,
+/area/ship/crew)
+"pa" = (
+/obj/machinery/photocopier/faxmachine/longrange,
+/obj/item/toy/figure/detective{
+	name = "Intel Buddy";
+	pixel_y = 10;
+	pixel_x = 5;
+	toysay = "Let's see what Command thinks of this..."
+	},
+/obj/structure/table/glass,
+/obj/effect/turf_decal/corner/opaque/syndiered/border{
+	dir = 4
+	},
+/turf/open/floor/plasteel/dark,
+/area/ship/cargo/office)
+"pM" = (
+/turf/closed/wall/mineral/plastitanium,
+/area/ship/cargo/office)
+"qa" = (
+/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4{
+	dir = 10
+	},
+/obj/structure/reagent_dispensers/fueltank,
+/obj/effect/turf_decal/box,
+/turf/open/floor/plating,
+/area/ship/engineering)
+"qf" = (
+/obj/machinery/door/poddoor{
+	id = "scbay"
+	},
+/obj/structure/cable,
+/obj/machinery/power/shieldwallgen/atmos/roundstart{
+	id = "scholo";
+	dir = 8
+	},
+/turf/open/floor/plating,
+/area/ship/cargo)
+"qo" = (
+/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2,
+/obj/structure/cable{
+	icon_state = "1-8"
+	},
+/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4{
+	dir = 9
+	},
+/turf/open/floor/plasteel/white,
+/area/ship/cargo/office)
+"qv" = (
+/obj/machinery/power/smes/shuttle/precharged{
+	dir = 4
+	},
+/obj/structure/window/reinforced{
+	dir = 8
+	},
+/obj/effect/turf_decal/trimline/opaque/syndiered/filled/line{
+	dir = 4
+	},
+/obj/structure/window/reinforced,
+/obj/structure/cable{
+	icon_state = "0-8"
+	},
+/obj/machinery/door/poddoor/shutters/preopen{
+	id = "scengine"
+	},
+/turf/open/floor/plating,
+/area/ship/engineering)
+"qz" = (
+/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2{
+	dir = 4
+	},
+/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4{
+	dir = 8
+	},
+/obj/structure/cable{
+	icon_state = "4-8"
+	},
+/obj/structure/window/reinforced{
+	dir = 4
+	},
+/obj/machinery/door/window/brigdoor,
+/obj/structure/window/reinforced{
+	dir = 8
+	},
+/obj/effect/turf_decal/corner/opaque/syndiered/bordercee,
+/obj/machinery/suit_storage_unit/inherit,
```

Comment:
Suit storage units can be given a req_access variable directly, so the reinforced glass enclosures around these aren't strictly necessary if you want to protect them. You also may want to move these SSUs into the cargo bay to avoid creating a bottleneck in the middle of the hallway.

Reply:
I'm fond of the little enclosures they're in. It really makes it feel like the thing is crammed full of gear.
![StrongDMM_t3BWMuiIv6](https://user-images.githubusercontent.com/94164348/185770610-77f5e171-9f50-4594-8ced-401e884f87dd.png)
 Is the only other spot I could think to put them without screwing up the very nice flow of the cargo bay.